tests/expectations/compiler/array/array_in_composite_data_types.out

```diff
@@ -11,7 +11,8 @@ function foo:
     input r0 as [[boolean; 8u32]; 8u32].private;
     cast 1u8 1u8 1u8 1u8 1u8 1u8 1u8 1u8 into r1 as [u8; 8u32];
     cast r1 into r2 as bar;
-    cast self.signer r1 into r3 as floo.record;
+    cast 1u8 1u8 1u8 1u8 1u8 1u8 1u8 1u8 into r3 as [u8; 8u32];
```

Comment:
This was a slightly unexpected change. It seems to now duplicate an instruction. Any idea why?